packages/wallet/package.json

```diff
@@ -9,8 +9,11 @@
   "author": "Horizon Blockchain Games",
   "license": "Apache-2.0",
   "scripts": {
-    "test": "yarn test:file tests/**/*.spec.ts",
+    "test": "yarn test:concurrently 'yarn test:run'",
+    "test:run": "yarn test:file tests/**/*.spec.ts",
     "test:file": "TS_NODE_PROJECT=../../tsconfig.test.json mocha -r ts-node/register --timeout 30000",
+    "test:concurrently": "concurrently -k --success first 'yarn start:hardhat_b > /dev/null'",
```

Comment:
why > /dev/null ..? also, why use concurrently when you're just starting a single service? perhaps more reliable cancellation of the process..?

Reply:
drops the second hardhat output, otherwise tests become impossible to read. Concurrently is to run hardhat2 and the tests at the same time, following the same pattern we use on the other packages.